docs/list-tested-APIs.md

```diff
@@ -0,0 +1,28 @@
+# Available APIs
+
+✅ Available and tested
+
+❗Warning: APIs not marked as tested are experimental. Use them at your own risk.
```

Comment:
We discussed this already in a call, but I would prefer calling them something else than experimental, as the service owner should already have tested the APIs.

Perhaps instead of calling the "untested" APIs experimental, we could call the "double tested" APIs `cross-validated` or something the like.